script.py

```diff
@@ -68,6 +73,30 @@ def choose_class():
     return class_name
 
 
+# This adds a spell to a list of already used spells
+def add_spell_to_list(chosen_spell):
+    file = open("used_spells.txt", "r")
+    file_info = file.read().split(',')
+    file.close()
+    if chosen_spell not in file_info:
```

Comment:
I get this when I try
`HIT ENTER TO BEGIN
You chose Bard

Teleport
 This spell instantly transports you and up to eight willing creatures of your choice that you can
see within range, or a single object that you can see within range, to a destination you select.  If
you target an object, it must be able to fit entirely inside a 10-foot cube, and it can’t be held or
carried by an unwilling creature.

Has This Spell Been Cast Before? [y/n]
y
Traceback (most recent call last):
  File "/Users/hpwilson/dnd_spells/python_scripts/script.py", line 128, in <module>
    add_spell_to_list(chosen_spell[1])
  File "/Users/hpwilson/dnd_spells/python_scripts/script.py", line 80, in add_spell_to_list
    file = open("used_spells.txt", "ra")
ValueError: must have exactly one of create/read/write/append mode`

Reply:
Ah sorted, it needs to be changed to append+ mode!